api/swagger.yaml

```diff
@@ -1859,6 +1859,13 @@ definitions:
                 type: "string"
                 x-nullable: false
                 example: "plugins.sock"
+              ProtocolScheme:
+                type: "string"
+                example: "some.protocol/v1.0"
```

Comment:
Should we use the actual scheme here? (`moby.plugins.http/v1`)? Are other schemes supported?

Reply:
I don't think we need to. The real accepted values are in the enum spec.